### ISSUE TITLE
Allow "Vertical" splitting of Query Editor window. 

### ISSUE THREAD

**Issue by CelsoDM** [DaxStudio/DaxStudio#1191]
**Is your feature request related to a problem? Please describe.**
When you are working on a large query in the query editor, it is tedious on large queries to scroll up and down all the time. Sometimes you want to refer back to an earlier section of the code, particularly, in the DEFINE block... 

**Describe the solution you'd like**
It may be helpful to have the ability to Split a window vertically as you can do in SQL Server Management Studio... (Window, Split, and Remove Split...)

**Describe alternatives you've considered**
Reducing the font size by holding the control key may help but it gets very small...

**Comment by dgosbell** [comment]
Thanks for posting this idea, but I do not believe the edit control we are using supports this sort of functionality, so I'm not sure if it is possible - I'll need to do some experimenting. Have you looked at the new [code folding](https://daxstudio.org/docs/features/indent-code-folding/) feature? Does that help at all when working with large queries?

**Comment by CelsoDM** [comment]
Absolutely! This code folding feature will work out great for me and is much better than simply splitting a window. This will save a bunch of time scrolling vertically. I can now collapse the DEFINE Block instantly and that’s pretty much what I needed to do.

Thanks for the reply. I will share this with the rest of our BI folks!
